tox.ini

```diff
@@ -1,9 +1,16 @@
 [tox]
 envlist =
-    py{27,34,35}-sphinx{13,14,15}
+    py{27,34,35,36}-sphinx{13,14,15}
```

Comment:
The original change just dropped 13 due to failing tests, we should instead ensure 1.3 support unless we are dictating which versions are deprecated. As far as I can tell, there is no reason not to support 1.3, it's just that state is stored in a different place.